api/base/utils.py

```diff
@@ -147,18 +147,15 @@ def default_node_list_queryset(model_cls):
     assert model_cls in {Node, Registration}
     return model_cls.objects.filter(is_deleted=False).annotate(region=F('addons_osfstorage_node_settings__region___id'))
 
-def default_node_permission_queryset(user, model_cls):
+def default_node_permission_queryset(auth, model_cls):
```

Comment:
making argument `auth` instead of user, so we can also take advantage of the private link that is stored on auth.